test/test_api/test_api.py

```diff
@@ -403,3 +403,8 @@ def test_infer_after_parentheses(Script, code, column, expected):
         assert completions == []
     else:
         assert [c.name for c in completions] == [expected]
+
+
+def test_infer_in_operator(Script):
+    completions = Script("res = 'f' in 'foo'").infer(column=1)
+    assert completions[0].full_name == 'builtins.bool'
```

Comment:
Can you please remove that test and add it to `test/completion/basic.py`? You can probably guess how that works in there.

Thanks a lot!